packages/storybook/src/nimble/theme-provider/tokens.stories.ts

```diff
@@ -1,198 +1,18 @@
-import type { Meta, StoryObj } from '@storybook/html';
-import { html, repeat, ViewTemplate, when } from '@microsoft/fast-element';
-import { waitForUpdatesAsync } from '../../../../nimble-components/src/testing/async-helpers';
-import { PropertyFormat } from '../../../../nimble-components/src/theme-provider/tests/types';
-import {
-    tokenNames,
-    cssPropertyFromTokenName,
-    scssPropertyFromTokenName,
-    TokenSuffix,
-    suffixFromTokenName
-} from '../../../../nimble-components/src/theme-provider/design-token-names';
-import { comments } from '../../../../nimble-components/src/theme-provider/design-token-comments';
-
-import {
-    bodyFont,
-    bodyFontColor,
-    groupHeaderFont,
-    groupHeaderFontColor,
-    groupHeaderTextTransform
-} from '../../../../nimble-components/src/theme-provider/design-tokens';
+import type { Meta } from '@storybook/html';
```

Comment:
chatted offline about parameterizing the stories based more on a normal story pattern instead of matrix story patterns, ie how appearance modes or status table is parameterized, etc.